queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND  '2019-01-01';
```

Comment:
- [ ] Kindly make sure that the end date should be 2019-31-12 as according to requirements it should display list of the names of all animals born between 2016 and 2019.
`SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';`

- [ ] Kindly make sure to add a screenshot for this query as in the pull request the query for this is not right.

Reply:
Makes a lot of sense, thanks. DONE